features/support/helpers.rb

```diff
@@ -149,13 +154,13 @@ def seconds_agnostic_datetime(datetime = Time.now)
     "#{time}:\\d{2}",
     Regexp.escape(zone)
   ] \
-  .join("\\ ")
+    .join("\\ ")
 end
 
 #
 
 def seconds_agnostic_time(time)
   time = time.strftime("%H:%M:%S") if time.is_a?(Time)
-  hour, minutes, _ = time.split(":")
+  hour, minutes, = time.split(":")
```

Comment:
I don't know about RC, but some tools allow named exceptions to a rule, something like:
```ruby
hour, minutes, ignore = time.split(":")
hour, minutes, ignore_seconds = time.split(":")
```

Also http://stackoverflow.com/a/33006942/253468